src/main/java/weblogic/logging/exporter/Startup.java

```diff
@@ -41,12 +45,26 @@ public static void main(String[] argv) {
       System.out.println("Reading configuration from file name: " + file.getAbsolutePath());
       Config config = Config.loadConfig(file);
       System.out.println(config);
+
+      //  Elastic log handler is enabled or the file log handler
       if (config.getEnabled()) {
         logger.addHandler(new LogExportHandler(config));
-      } else {
-        System.out.println("WebLogic Logging Exporter is disabled");
+      } else if (config.isFileLoggingEnabled()) {
+        // Because of this bridge log messages in the applications themselves are being forwarded to the server logger.
+        // so that logging in ear/war artifacts are also visible to the server logger and appear in the JSON log file.
+        Logger.getLogger("").addHandler(new ServerLoggingHandler());
+
+        // Register a file handler using the provided config
+        FileHandler fh = new FileHandler(config.getOutputFile(), config.getMaxFileSize(), config.getGetMaxRollbackFiles(), config.getAppendToFile());
+        fh.setLevel(Level.parse(config.getFileLoggingLogLevel()));
+        fh.setFormatter(new EcsFormatter());
+        logger.addHandler(fh);
+       } else {
+        System.out.println("WebLogic Elasticsearch Logging Exporter is disabled");
```

Comment:
@russgold, @marinakog, I'm not as familiar with the logging exporter code base. I'm guessing that we are using System.out.println for logging from the exporter because the Java logging system is being used for messages bridged from WebLogic, correct? Is it possible to structure this so that we can still use logging for the exporter's messages?

Reply:
I'm not familiar with this code base at all, actually. And Huiling is the one who was testing this exporter, not Marina.